lib/jira/client/index.js

```diff
@@ -126,16 +126,20 @@ module.exports = async (jiraHost, gitHubInstallationId, logger) => {
           if (!withinIssueKeyLimit(data.commits) || !withinIssueKeyLimit(data.branches)) {
             truncateIssueKeys(data)
             const subscription = await Subscription.getSingleInstallation(jiraHost, gitHubInstallationId)
-            await subscription.update({syncWarning: 'Exceeded issue key reference limit. Some issues may not be linked.'})
+            await subscription.update({ syncWarning: 'Exceeded issue key reference limit. Some issues may not be linked.' })
           }
 
-          await instance.post('/rest/devinfo/0.10/bulk', {
-            preventTransitions: (options && options.preventTransitions) || false,
-            repositories: [data],
-            properties: {
-              installationId: gitHubInstallationId
-            }
-          })
+          if (data.commits && data.commits.length >= 400) {
```

Comment:
I would recommend putting the `400` in a constant somewhere, otherwise it's a magic comment with only this PR as reference as to what it means :)